CSSintroAss.html

```diff
@@ -0,0 +1,92 @@
+          <p id "combSteps" class "content">Using an icecream machine, pour liquids into the frozen basinand proccess according to manufacturer instructions, i.e. let spin and thicken for 20 minutes before adding, mascapone, fig mixture and the nuts. Continue to proccess for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving.</p>
+      <section class="groceryList" class="clearfix">
+          <section class ="groceries grid_7">
+              <input type='checkbox' id='list1' value='1 1/2 cups milk'/>
+              <label for='list1'>1 1/2 cups milk</label><br>
+              <input type='checkbox' id='list2' value='1/2 cup mascarpone'/>
+              <label for='list2'>1/2 cup mascarpone</label><br>
+              <input type='checkbox' id='list3' value='1/2 tsp pink salt'/>
+              <label for='list3'>1/2 tsp pink salt</label><br>
+              <input type='checkbox' id='list4' value='1 lb Black Mission Figs'/>
+              <label for='list4'>1 lb Black Mission Figs</label><br>
+              <input type='checkbox' id='list5' value='1/2 cup brown sugar'/>
+              <label for='list5'>1/2 cup brown sugar</label><br>
+              <input type='checkbox' id='list6' value='2-4 tbsp water'/>
+              <label for='list6'>2-4 tbsp water</label>
+          </section>  
+          <section class ="groceries">
+              <input type='checkbox' id='list7' value='1 1/2 cups heavy cream'/>
+              <label for='list7'>1 1/2 cups heavy cream</label><br>
+              <input type='checkbox' id='list8' value='1/3 granulated sugar'/>
+              <label for='list8'>1/3 granulated sugar</label><br>
+              <input type='checkbox' id='list9' value='2 egg yolks'/>
+              <label for='list9'>2 egg yolks</label><br>                              
+              <input type='checkbox' id='list1' value='10 lemon, juiced'/>
+              <label for='list10'>1 lemon, juiced</label><br>
+              <input type='checkbox' id='list11' value='2 tbsp butter'/>
+              <label for='list11'>2 tbsp butter</label><br>
+              <input type='checkbox' id='list12' value='1 cup honey roasted pecans, roughly chopped'/>
+              <label for='list12'>1 cup honey roasted pecans, roughly chopped</label>
+          </section> 
+      </section>
+      </article>
+      <article class="clearfix">
+        <img class="grid_2" src="images/van-pic.png">
+      <section class="grid_8">
+        <h3>Vanessa Stevenson</h3>
+        <p>Food enthusiast, photography fan. Add a pinch of raw foodism and that's pretty much who I am.</p >
+      </section>
+        <button class="grid_3 push_3">Share Recipe</button>
+      </article>
+      <hr class='bottomline'>
```

Comment:
We don't use hr tags because in most case they're HTML that is affecting style. Use CSS to achieve this effect.